src/jumpstart/JumpstartEnterAmount.tsx

```diff
@@ -167,10 +175,38 @@ function JumpstartEnterAmount() {
     jumpstartSendStatus === 'success' ||
     jumpstartSendStatus === 'loading'
 
+  const onIntroDismiss = () => {
+    dispatch(jumpstartIntroSeen())
+    replace(Screens.JumpstartEnterAmount)
```

Comment:
what happens if we don't have this `replace`? maybe we don't need it? i'm guessing this is here to "fake" a navigation animation, if it needs to be here maybe we can add a comment so that we remember why we need it?